pkg/test/helpers/grpc_health.go

```diff
@@ -32,15 +32,15 @@ import (
 var errServiceUnhealthy = errors.New("service is unhealthy")
 
 // HealthCheck returns a function for ginkgo "Eventually" poll it repeatedly to check whether a gRPC server is ready.
-func HealthCheck(addr string, connTimeout time.Duration, rpcTimeout time.Duration, opts ...grpc.DialOption) func() error {
+func HealthCheck(disableLogging bool, addr string, connTimeout time.Duration, rpcTimeout time.Duration, opts ...grpc.DialOption) func() error {
```

Comment:
I believe we should not disable logging as it helps us identify why health checks fail.

Reply:
We discussed and agreed that the help command info should be disabled when an error occurs. The regular logging messages should still be displayed.